gui/default/assets/css/theme.css

```diff
@@ -7,29 +7,5 @@
 
 */
 
-.panel-progress {
-    background: #3498db;
-}
-
-.identicon rect {
-    fill: #333;
-}
-
-.panel-warning .identicon rect {
-    fill: #fff;
-}
-
-.li-column {
-    background-color: rgb(236, 240, 241);
-    border-radius: 3px;
-}
-
-.panel-heading:hover, .panel-heading:focus {
-    text-decoration: none;
-}
-
-.fancytree-ext-filter-hide tr.fancytree-submatch span.fancytree-title,
-.fancytree-ext-filter-hide span.fancytree-node.fancytree-submatch span.fancytree-title {
-    color: black !important;
-    font-weight: lighter !important;
-}
+@import "/dark/assets/css/theme.css" (prefers-color-scheme: dark);
```

Comment:
I don't like this path, this should either be be assets/<theme>/<stuff>
or 
themes/<theme>/assets/<stuff>
or 
/assets/css/theme.css?theme=dark (this is probably not very cache friendly).
or something like that. 

Random stuff rooted feels sad.

Reply:
I am not sure if I understand what path you want in the first two cases.
For example, in the second option, if the path is not rooted, as in:
```css
@import "theme-assets/dark/assets/css/theme.css" (...);
```
the browser would request `/assets/css/theme-assets/dark/assets/css/theme.css` to the server, so I guess I would have to exclude any path that has `theme-assets` in it to handle it. Is that what you want?

Reply:
Sorry, what I mean is that I do not want localhost:8483/dark/foo/bar, as /dark is at the root of webserver, does not describe that is it, or what it represents.

Reply:
We could have 
/theme-specific-assets/dark/css/theme.css

Reply:
Okay, I see now what you meant. Thank you for clarifying!